src/main/java/mServer/crawler/sender/MediathekArd.java

```diff
@@ -105,7 +106,7 @@ private void addTage() {
               thema = thema.substring(0, thema.indexOf(','));
             }
             datum = seite1
-                .extract("<title>Videos (TV-Sendungen) des Senders Das Erste vom", "- ARD").trim();
+                    .extract("<title>Videos (TV-Sendungen) des Senders Das Erste vom", "- ARD").trim();
             titel = seite1.extract("<h4 class=\"headline\">", "<", pos);
             dauer = seite1.extract("<p class=\"subtitle\">", "<", pos);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "<p class=\"subtitle\">" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)